app/models/identity.rb

```diff
@@ -4,6 +4,7 @@ class Identity < ApplicationRecord
   enum provider: %i[
     twitter
     facebook
+    identity
```

Comment:
Having a type called "identity" in a class called "identity" feels like a recipe for confusion, could we rename this to email or devise or something?

Reply:
Done, changed to devise_email. This doesn't actually seem to be used anywhere.